modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -29,6 +29,100 @@ public class UpgradeCatchAllCheck {
 		).getCode();
 	}
 
+	public void method(
+		long groupId, long userId, Map<Locale, String> nameMap,
+		Map<Locale, String> shortDescriptionMap,
+		Map<Locale, String> descriptionMap, Map<Locale, String> urlTitleMap,
+		Map<Locale, String> metaTitleMap,
+		Map<Locale, String> metaDescriptionMap,
+		Map<Locale, String> metaKeywordsMap, String productTypeName,
+		boolean ignoreSKUCombinations, boolean shippable, boolean freeShipping,
+		boolean shipSeparately, double shippingExtraPrice, double width,
+		double height, double depth, double weight, long cpTaxCategoryId,
+		boolean taxExempt, boolean telcoOrElectronics, String ddmStructureKey,
+		boolean published, int displayDateMonth, int displayDateDay,
+		int displayDateYear, int displayDateHour, int displayDateMinute,
+		int expirationDateMonth, int expirationDateDay, int expirationDateYear,
+		int expirationDateHour, int expirationDateMinute, boolean neverExpire,
+		String defaultSku, boolean subscriptionEnabled, int subscriptionLength,
+		String subscriptionType,
+		UnicodeProperties subscriptionTypeSettingsUnicodeProperties,
+		long maxSubscriptionCycles, String externalReferenceCode,
+		ServiceContext serviceContext) {
+
+		CPDefinition cpDefinition1 =
+			_cpDefinitionLocalService.upsertCPDefinition(
+				groupId, userId, nameMap, shortDescriptionMap, descriptionMap,
+				urlTitleMap, metaTitleMap, metaDescriptionMap, metaKeywordsMap,
+				productTypeName, ignoreSKUCombinations, shippable, freeShipping,
+				shipSeparately, shippingExtraPrice, width, height, depth,
+				weight, cpTaxCategoryId, taxExempt, telcoOrElectronics,
+				ddmStructureKey, published, displayDateMonth, displayDateDay,
+				displayDateYear, displayDateHour, displayDateMinute,
+				expirationDateMonth, expirationDateDay, expirationDateYear,
+				expirationDateHour, expirationDateMinute, neverExpire,
+				defaultSku, subscriptionEnabled, subscriptionLength,
+				subscriptionType, subscriptionTypeSettingsUnicodeProperties,
+				maxSubscriptionCycles, externalReferenceCode, serviceContext);
+
+		CPDefinition cpDefinition2 =
+			CPDefinitionLocalServiceUtil.upsertCPDefinition(
+				groupId, userId, nameMap, shortDescriptionMap, descriptionMap,
+				urlTitleMap, metaTitleMap, metaDescriptionMap, metaKeywordsMap,
+				productTypeName, ignoreSKUCombinations, shippable, freeShipping,
+				shipSeparately, shippingExtraPrice, width, height, depth,
+				weight, cpTaxCategoryId, taxExempt, telcoOrElectronics,
+				ddmStructureKey, published, displayDateMonth, displayDateDay,
+				displayDateYear, displayDateHour, displayDateMinute,
+				expirationDateMonth, expirationDateDay, expirationDateYear,
+				expirationDateHour, expirationDateMinute, neverExpire,
+				defaultSku, subscriptionEnabled, subscriptionLength,
+				subscriptionType, subscriptionTypeSettingsUnicodeProperties,
+				maxSubscriptionCycles, externalReferenceCode, serviceContext);
```

Comment:
@TamyrisTorres the previous cases we didn't create a variable on the test. To follow a pattern we should remove this variable declaration and only leave it when is necessary for the pattern.